BRAD/justchat.py

```
@@ -0,0 +1,130 @@
+from langchain_community.document_loaders import PyPDFLoader
+from langchain_community.document_loaders import TextLoader
+from langchain_community.embeddings.sentence_transformer import (
+    SentenceTransformerEmbeddings,
+)
+from sklearn.metrics.pairwise import cosine_similarity
+from langchain_text_splitters import CharacterTextSplitter
+from langchain.retrievers.multi_query import MultiQueryRetriever
+from langchain_community.callbacks import get_openai_callback
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+
+from langchain_text_splitters import RecursiveCharacterTextSplitter
```

Ruff

Redefinition of unused `RecursiveCharacterTextSplitter` from line 38

Remove definition: `RecursiveCharacterTextSplitter`

(F811)